### PR TITLE
[12.0] [MIG] pos_stock_picking_invoice_link: Migration to v12

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT25241

This module links stock moves and pickings generated on POS to its corresponding invoice lines. You can see an example on next image:

image

Please @chienandalu @ernestotejeda can you review this?

### CODE REVIEW

File: pos_stock_picking_invoice_link/README.rst [hunk 1]
```diff
@@ -14,13 +14,13 @@ POS Stock Picking Invoice Link
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpos-lightgray.png?logo=github
-    :target: https://github.com/OCA/pos/tree/11.0/pos_stock_picking_invoice_link
+    :target: https://github.com/OCA/pos/tree/13.0/pos_stock_picking_invoice_link
```
Comment (ernestotejeda) [inline pos_stock_picking_invoice_link/README.rst]: Regenerate readme for 12.0 version

**Review by ernestotejeda — COMMENTED** [review]
Code review :+1:

**Comment by CarlosRoca13** [comment]
@ernestotejeda change done!

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": ""
+}
```
Comment (chienandalu) [inline .vscode/settings.json]: careful, your editor config file is getting into the commit

**Review by chienandalu — CHANGES_REQUESTED** [review]
Tiny minor change

**Comment by CarlosRoca13** [comment]
@chienandalu change done

**Review by victoralmau — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu please check latest changes

**Review by chienandalu — APPROVED** [review]

**Pull request merged**